python/cugraph/layout/force_atlas2.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#aLicensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cugraph.layout import force_atlas2_wrapper
+from cugraph.structure.graph import Graph, null_check
+
+def force_atlas2(input_graph,
+        max_iter=500,
+        pos_list=None,
+        outbound_attraction_distribution=True,
+        lin_log_mode=False,
+        prevent_overlapping=False,
+        edge_weight_influence=1.0,
+        jitter_tolerance=1.0,
+        barnes_hut_optimize=True,
+        barnes_hut_theta=0.5,
+        scaling_ratio=2.0,
+        strong_gravity_mode=False,
+        gravity=1.0,
+        verbose=False,
+	callback=None):
+
+        """
+        ForceAtlas2 is a continuous graph layout algorithm for handy network
+        visualization.
+
+        Parameters
+        ----------
+        input_graph : cugraph.Graph
+            cuGraph graph descriptor, should contain the connectivity information
+            as an edge list.
+            The adjacency list will be computed if not already present. The graph
+            should be undirected where an undirected edge is represented by a
+            directed edge in both direction.
+
+        max_iter : integer
+            This controls the maximum number of levels/iterations of the Force Atlas
+            algorithm. When specified the algorithm will terminate after no more
+            than the specified number of iterations. No error occurs when the
+            algorithm terminates early in this manner.
+        pos_list: cudf.DataFrame
+            Data frame with initial positions containing two columns:
```

Comment:
Data frame with initial positions -> Data frame with initial **vertex** positions

Also, how do you know what 'x' and 'y' correspond to after the graph is renumbered? We can't assume that it implicitly matches internal data. Should we accept "vetrex_id","x","y"?

Reply:
Indeed we should add it.